lab3.md

```diff
@@ -34,19 +34,22 @@ After that, to find out which languages are available type in:
 $ faas-cli new --list
 Languages available as templates:
 - csharp
+- csharp-armhf
 - dockerfile
+- dockerfile-armhf
 - go
 - go-armhf
+- java12
+- java8
 - node
 - node-arm64
 - node-armhf
+- php7
 - python
 - python-armhf
 - python3
+- python3-armhf
 - ruby
-
-Or alternatively create a folder containing a Dockerfile, then pick
```

Comment:
Can you change this to say that they can use the Dockerfile template for something completely custom?

Reply:
Should this comment be a statement in the workshop or actually returned by the `faas-cli new --list` command? 🤔